Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -180,6 +180,12 @@ class BookmarksViewController: SiteTableViewController {
       print(error.description)
     }
     
+    if tableView.isEditing == false && bookmarksFRC?.fetchedObjects?.count == 0 {
+        editBookmarksButton?.isEnabled = false
+    } else {
+        editBookmarksButton?.isEnabled = true
```

Comment:
`editBookmarksButton` is forced unwrapped in the declaration so `?` is not necessary.

Reply:
Build succeeds, however when you try pressing menu button throws:

> Thread 1: Fatal error: Unexpectedly found nil while unwrapping an Optional value